src/frontend/boilerplates/archipelago/public/index.html

```diff
@@ -11,6 +11,8 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" crossorigin="" />
```

Comment:
pas moyen de l'embarquer dans un composant?

Reply:
Possible mais pas vraiment une bonne pratique.
J'ai noté qu'il fallait le faire dans le README du package.